Symfony/CS/Fixer/Symfony/SpacesAfterSemicolonFixer.php

```diff
@@ -49,4 +56,52 @@ public function getDescription()
     {
         return 'Fix whitespace after a semicolon.';
     }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     *
+     * @return int 0|1 number of tokens inserted
+     */
+    private function ensureSingleSpaceAfter(Tokens $tokens, $index)
```

Comment:
TODO: rename, since it doesn't ensure a single space, it doesn't  touch multi line whitespaces (as it should, i.e. logic correct, naming confusing)